CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - SC-1402 Replace usage of lazyLoad by lazyLoadComponent in Select
 - SONAR-12679 Introduce new BranchIcon
 - SONAR-12633 New BackIcon
+- SONAR-12637 Allow ActionsDropdown to be positioned like any other dropdown
```

Comment:
You might not be up to date with master ?
We are at version 41 I think

Reply:
Ah, thought I had pulled. Will do.